src/utils/camelCase.test.js

```diff
@@ -0,0 +1,5 @@
+import camelCase from "./camelCase";
+
+test("Converts string to camelCase", () => {
+  expect(camelCase("My_amazing_project-1")).toBe("myAmazingProject1");
+});
```

Comment:
This looks great Lois but it looks like we also use an npm package called [camelcase](https://www.npmjs.com/package/camelcase) that might do something similar

Reply:
Ah this had completely passed me by 😅 Thanks, I've switched to use the library version instead 👍